backend/src/packages/chat-messages/chat-message.model.ts

```diff
@@ -0,0 +1,35 @@
+import { Model, type RelationMappings } from 'objection';
+
+import {
+  AbstractModel,
+  DatabaseTableName,
+} from '#libs/packages/database/database.js';
+import { UsersTableColumn } from '#packages/users/libs/enums/enums.js';
+import { UserDetailsModel } from '#packages/users/user-details.model.js';
+
+import { ChatMessagesTableColumn } from './libs/enums/enums.js';
+
+class ChatMessageModel extends AbstractModel {
+  public message!: string;
+  public chatId!: number;
+  public senderId!: number;
```

Comment:
Lets leave black line after each field

Reply:
no need to fix this in this pr, I will add linter rule for this, to prevent this cases

Reply:
I thought we had the linter rule for this, cause I had a message that we need to have a blank line  🤷‍♂️

Reply:
maybe it's my personal rule then 🤔😆